### DISCUSSION TITLE
Can Rectangle be put on the App Store

### DISCUSSION THREAD

**Discussion by AlexandruIstrate** [rxhanson/Rectangle#453] [Q&A] [answered]
It would be nice for the app to be on the App Store for obvious reasons (updates, easier distribution). Also the App Store is the Apple way of doing things and I would like to see this app there.

**Comment by rxhanson** [comment, marked as answer]
I agree. Unfortunately, new window management applications are no longer allowed on the App Store since they cannot be sandboxed.

**Reply by AlexandruIstrate** [reply]
how does Magnet manage to be on the App Store then?

**Reply by rxhanson** [reply]
Magnet, Moom, and BetterSnapTool got in while it was still allowed, many years ago.

**Reply by AlexandruIstrate** [reply]
fair. didn't know that

**Reply by AlexandruIstrate** [reply]
That solves it. I don't know how to close the discussion though. So @rxhanson you can do that if you want to.
